tests/php/Security/MemberTest.php

```diff
@@ -1831,7 +1831,7 @@ public function testMapInCMSGroups(array $groupFixtures, array $groupCodes, arra
         $result = Member::mapInCMSGroups($groups);
         $this->assertInstanceOf(Map::class, $result);
 
-        $this->assertSame($expectedUsers, $result->keys());
+        $this->assertEqualsCanonicalizing($expectedUsers, $result->keys());
```

Comment:
In PostgreSQL, the result was returning the keys in a slightly different order.

The order of the results is irrelevant here. `assertEqualsCanonicalizing' sorts the arrays before comparing them.